packages/calcite-components/src/components/list/list.e2e.ts

```diff
@@ -339,4 +339,87 @@ describe("calcite-list", () => {
     expect(await listItemOne.getProperty("selected")).toBe(false);
     expect(await list.getProperty("selectedItems")).toHaveLength(0);
   });
+
+  describe.only("keyboard navigation", () => {
```

Comment:
Can you remove `only` here? The linter should've caught this. 🤔

Reply:
eek. Hmm my linter isn't catching this. No bueno!

Maybe because its on the describe???

Reply:
Hmmm nope. No linting going on with `.only` for me. Does it work for you?

Reply:
Weird. Looks like the linter does catch it, but the commit hook isn't being blocked. Something going on with our `lint-staged` configuration? cc @benelan